typings/config.d.ts

```diff
@@ -65,6 +107,18 @@ declare global {
         instance: InstanceType<typeof Gatherer>;
         options: {};
       }
+
+      export interface AuditDefn {
+        implementation: typeof Audit;
+        options: {};
+      }
+
+      // For now, these are unchanged from JSON version
+      export interface CategoryMember extends CategoryMemberJson {}
```

Comment:
in addition to the doubled-up type (`LH.Result.Category.CategoryMember`), this has two names?